annotations/src/main/java/org/robolectric/annotation/ClassName.java

```diff
@@ -6,29 +6,33 @@
 import java.lang.annotation.Target;
 
 /**
- * Parameters with types that can't be resolved at compile time may be annotated @ClassName.
+ * Parameters, function return with types that can't be resolved at compile time may be annotated
+ * with @ClassName.
  *
  * <p>Use this annotation when creating shadow methods that contain new Android types in the method
- * signature that do not exist in older SDK levels.
+ * signature, return type that do not exist in older SDK levels.
  *
  * <pre>
  * &#64;Implements(FooAndroidClass.class)
  * class ShadowFooAndroidClass {
  *
  *    // A method shadowing FooAndroidClass#setBar(com.android.RealClassName, int, String)
  *    &#64;Implementation
- *    public void setBar(&#64;ClassName("com.android.RealClassName") Object para1, int para2, String para3) {
+ *    public &#64;ClassName("com.android.RealReturnType") Object setBar(
+ *        &#64;ClassName("com.android.RealClassName") Object para1,
+ *        int para2,
+ *        String para3) {
  *
  *    }
  * }
  * </pre>
  */
-@Target(ElementType.PARAMETER)
+@Target({ElementType.PARAMETER, ElementType.TYPE_USE})
 @Retention(RetentionPolicy.RUNTIME)
 public @interface ClassName {
 
   /**
-   * The class name intended for this parameter.
+   * The class name intended for the parameter, function return.
```

Comment:
`The class name intended for the parameter or the function return value.`

Reply:
done